### DISCUSSION TITLE
Static food source?

### DISCUSSION THREAD

**Discussion by SelfSealShelf** [MaxRobinsonTheGreat/EvolutionSimulatorV2#58] [Ideas and Suggestions]
There's no way to create a static food source currently.

**Comment by syspic** [comment]
just a note in case you didnt see it: you can create some with your mouse. you can also have some randomly spawn in the world. I understand if your use case isnt one of those though!

**Comment by SelfSealShelf** [comment]
Right now I just used an AHK script to spam clicking. It doesn't work well

I'd like to be able to shape static food sources around walls to evolve certain mouth shapes

**Comment by MaxRobinsonTheGreat** [comment]
Moved to discussions. Please don't make issues for feature requests, and I'm not sure what you mean by static food source

**Reply by syspic** [reply]
I believe they mean a way to replenish food. So perhaps there is a food cell type that comes back after X seconds. or maybe it gets "eaten" but does not go away. Like a fountain of food lol
